src/colour/web/index.md

```diff
@@ -120,4 +120,4 @@ Use for success messages
 Indicative examples for illustrative purposes only.
 {% endcallout %}
 
-![TODO](./example.png)
+![A screenshot of the GOV.UK homepage on desktop, showing web palette colours such as brand colour blue for the header, link colour blue for the links and visited purple for visited links.](./example.png)
```

Comment:
Is there a reason why you have used alternative names of the colours? To make people understand what the colours are exactly, it makes sense to name them the same way they are named on the page. Either Primary blue and Primary purple, or $govuk-link-colour and $govuk-link-visited-colour.

Reply:
I referred to the web palette names, as I was thinking this would encourage people to use the web colour "variables" (not sure that's the right term). 

Naming these using the proper names could work, as long as it's understood they're the same colour with identical values... 🤔